src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+$cell-size: 100px;
```

Comment:
Consider using a more descriptive variable name like `$calendar-cell-size` to improve readability.